src/scripts/main.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
+const body = document.querySelector('body');
+
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+  const header = document.createElement('h2');
+  const paragraph = document.createElement('p');
+
+  body.append(notification);
+  notification.classList.value = `notification ${type}`;
+  notification.style.top = posTop + 'px';
+  notification.style.right = posRight + 'px';
```

Comment:
to be more consistent you can apply string templates on lines 12 and 13 as well as on line 11